test/controllers/CheckYourAppealController.test.ts

```diff
@@ -210,17 +241,22 @@ describe('CheckYourAppealController', () => {
                 'xUHinh19D17SQV2BYRLnGEZgeovYhcVitzLJMxpGxXW0w+30EYBb+6yF44pDWPsPejI17R5JSwy/Cw5kYQKO2A==';
 
             const appealsService = createSubstituteOf<AppealsService>(service => {
-                service.save(Arg.any(), Arg.any(), Arg.any()).returns(Promise.resolve(Arg.any()));
+                service.save(Arg.any(), Arg.any(), Arg.any()).returns(Promise.resolve('1'));
```

Comment:
Why did you change this to 1?

Reply:
Substitute was having a hard time comparing Arg.any() when we do 
![image](https://user-images.githubusercontent.com/54070318/85296268-0976d480-b499-11ea-8107-54faf11ec3a8.png)

Reply:
The save method was saying that the args didn't match because { id: Arg.any() } did not equal { id: Arg.any() } :/